tests/tree_xml_parser_reg_text_tests.py

```diff
@@ -205,6 +205,135 @@ def test_build_from_section_table(self):
+          <SECTION>
+            <SECTNO>§ 555.219</SECTNO>
+            <SUBJECT>%s</SUBJECT>
+            <EXTRACT>
+                <HD SOURCE="HD1">%s</HD>
+                <GPOTABLE CDEF="6.1,6.1,5.2,5.2,5.2" COLS="5" OPTS="L2">
+                  <BOXHD>
+                    <CHED H="1">Pounds</CHED>
+                    <CHED H="2">Over</CHED>
+                    <CHED H="2">Not over</CHED>
+                    <CHED H="1">From inhabited building distance (feet)</CHED>
+                    <CHED H="1">From public railroad and highway distance
+                        (feet)</CHED>
+                    <CHED H="1">From above ground magazine (feet)</CHED>
+                  </BOXHD>
+                  <ROW>
+                    <ENT I="01">0</ENT>
+                    <ENT>1,000</ENT>
+                    <ENT>75</ENT>
+                    <ENT>75</ENT>
+                    <ENT>50</ENT>
+                  </ROW>
+                  <ROW>
+                    <ENT I="01">1,000</ENT>
+                    <ENT>5,000</ENT>
+                    <ENT>115</ENT>
+                    <ENT>115</ENT>
+                    <ENT>75</ENT>
+                  </ROW>
+                </GPOTABLE>
+                <HD SOURCE="HD1">%s</HD>
+            </EXTRACT>
+          </SECTION>
+        """ % (subject, table_first_header_text, table_second_header_text))
+        nodes = reg_text.build_from_section('555', xml)
+        node = nodes[0]
+        self.assertEqual(u'§ 555.219 %s' % subject, node.title)
+        self.assertEqual('regtext', node.node_type)
+        self.assertEqual(['555', '219'], node.label)
+        self.assertEqual(1, len(node.children))
+
+        extract_node = node.children[0]
+        self.assertEqual('extract', extract_node.node_type)
+        self.assertEqual(['555', '219', 'p1'], extract_node.label)
+        self.assertEqual(3, len(extract_node.children))
+
+        first_header_node = extract_node.children[0]
+        self.assertEqual('regtext', first_header_node.node_type)
+        self.assertEqual('', first_header_node.text)
+        self.assertEqual(table_first_header_text, first_header_node.title)
+        self.assertEqual(['555', '219', 'p1', 'p1'], first_header_node.label)
+        self.assertEqual(0, len(first_header_node.children))
+
+        table_node = extract_node.children[1]
+        self.assertEqual('regtext', table_node.node_type)
+        self.assertEqual('GPOTABLE', table_node.source_xml.tag)
+        self.assertEqual(['555', '219', 'p1', 'p2'], table_node.label)
+        self.assertEqual(0, len(table_node.children))
+
+        second_header_node = extract_node.children[2]
+        self.assertEqual('regtext', second_header_node.node_type)
+        self.assertEqual('', second_header_node.text)
+        self.assertEqual(table_second_header_text, second_header_node.title)
+        self.assertEqual(['555', '219', 'p1', 'p3'], second_header_node.label)
+        self.assertEqual(0, len(second_header_node.children))
```

Comment:
Not at all necessary, but you may find a few libraries helpful in writing these kinds of tests.

`XMLBuilderMixin` might make the XML a little more concise (this is the `with self.tree` syntax) and `NodeAccessorMixin` might make it easier to drill down into the tree, e.g.
```python
section = self.node_accessor(nodes[0], ['555', '219'])
self.assertEqual(section['p1']['p3'].title, table_second_header_text)
```
As a wider point, it might make sense to create a `match` function, similar to https://github.com/18F/regulations-parser/blob/9d67464e95e0678e5a5edefc127cd2ef26864f72/tests/grammar_amdpar_tests.py#L349-L353 so that we could say something like:
```python
self.assert_attrib_match(section['p1']['p3'],
                         title=table_second_header_text,
                         node_type=Node.REGTEXT,
                         source_xml__tag='GPOTABLE')
```

Just pining now.